cmd/snap-update-ns/bootstrap.c

```diff
@@ -247,6 +247,91 @@ int validate_snap_name(const char* snap_name)
     return 0;
 }
 
+static int instance_key_validate(const char *instance_key)
```

Comment:
I wonder why we use "int" here instead of bool even though we include stdbool.h - I guess because validate_snap_name also uses int but that raises the same question. Anyway, probably fine for now.

Reply:
Maybe because of the cgo bridge code - ignore me here please.